src/main/java/com/exadel/discountwebapp/history/vo/HistoryRequestVO.java

```diff
@@ -0,0 +1,16 @@
+package com.exadel.discountwebapp.history.vo;
+
+import com.exadel.discountwebapp.history.enums.HistoryEnum;
+import com.exadel.discountwebapp.history.enums.OperationsEnum;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class HistoryRequestVO {
+    HistoryEnum entity;
+    Long id;
+    OperationsEnum operationType;
```

Comment:
No need to add "...Enum" postfix in enums

Reply:
ok